packages/flutter/lib/src/material/slider_theme.dart

```diff
@@ -0,0 +1,729 @@
+        otherData.valueIndicatorShape == valueIndicatorShape &&
+        otherData.showValueIndicator == showValueIndicator;
+  }
+
+  @override
+  void debugFillProperties(DiagnosticPropertiesBuilder description) {
+    super.debugFillProperties(description);
+    final ThemeData defaultTheme = new ThemeData.fallback();
+    final SliderThemeData defaultData = new SliderThemeData.materialDefaults(
+      primaryColor: defaultTheme.primaryColor,
+      primaryColorDark: defaultTheme.primaryColorDark,
+      primaryColorLight: defaultTheme.primaryColorLight,
+    );
+    description.add(new DiagnosticsProperty<Color>('activeRailColor', activeRailColor,
+        defaultValue: defaultData.activeRailColor));
+    description.add(new DiagnosticsProperty<Color>('inactiveRailColor', inactiveRailColor,
+        defaultValue: defaultData.inactiveRailColor));
+    description.add(new DiagnosticsProperty<Color>(
+        'disabledActiveRailColor', disabledActiveRailColor,
+        defaultValue: defaultData.disabledActiveRailColor));
+    description.add(new DiagnosticsProperty<Color>(
+        'disabledInactiveRailColor', disabledInactiveRailColor,
+        defaultValue: defaultData.disabledInactiveRailColor));
+    description.add(new DiagnosticsProperty<Color>('activeTickMarkColor', activeTickMarkColor,
+        defaultValue: defaultData.activeTickMarkColor));
+    description.add(new DiagnosticsProperty<Color>('inactiveTickMarkColor', inactiveTickMarkColor,
+        defaultValue: defaultData.inactiveTickMarkColor));
+    description.add(new DiagnosticsProperty<Color>(
+        'disabledActiveTickMarkColor', disabledActiveTickMarkColor,
+        defaultValue: defaultData.disabledActiveTickMarkColor));
+    description.add(new DiagnosticsProperty<Color>(
+        'disabledInactiveTickMarkColor', disabledInactiveTickMarkColor,
+        defaultValue: defaultData.disabledInactiveTickMarkColor));
+    description.add(new DiagnosticsProperty<Color>('thumbColor', thumbColor,
+        defaultValue: defaultData.thumbColor));
+    description.add(new DiagnosticsProperty<Color>('disabledThumbColor', disabledThumbColor,
+        defaultValue: defaultData.disabledThumbColor));
+    description.add(new DiagnosticsProperty<Color>('overlayColor', overlayColor,
+        defaultValue: defaultData.overlayColor));
+    description.add(new DiagnosticsProperty<Color>('valueIndicatorColor', valueIndicatorColor,
+        defaultValue: defaultData.valueIndicatorColor));
+    description.add(new DiagnosticsProperty<SliderComponentShape>('thumbShape', thumbShape,
+        defaultValue: defaultData.thumbShape));
+    description.add(new DiagnosticsProperty<SliderComponentShape>(
+        'valueIndicatorShape', valueIndicatorShape,
+        defaultValue: defaultData.valueIndicatorShape));
+    description.add(new DiagnosticsProperty<ShowValueIndicator>(
+        'showValueIndicator', showValueIndicator,
+        defaultValue: defaultData.showValueIndicator));
+  }
```

Comment:
these are inconsistently wrapped which is weird

Reply:
I've already fixed this in my other PR.